CHANGELOG.md

```diff
@@ -32,6 +32,9 @@ Full documentation for hipSOLVER is available at [hipsolver.readthedocs.io](http
 ### Deprecated
 ### Removed
 ### Fixed
+- bufferSize functions will now return HIPSOLVER_STATUS_NOT_INITIALIZED instead of HIPSOLVER_STATUS_INVALID_VALUE when both handle and lwork are null.
+- Fixed rare memory allocation failure in syevd/heevd and sygvd/hegvd when using rocblas_device_malloc to allocate multiple workspace arrays.
```

Comment:
maybe better not to mention rocblas_device_malloc...?
This is an internal interface and it is not documented/supported anywhere (not for public use).
I would say:
"Fixed rare memory allocation failure in syevd/heevd and sygvd/hegvd when allocating multiple workspace arrays if the E parameter is null"
or something like this...